react/tic-tac-toe/src/components/Game/components/Board/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Square from './components/Square';
+import styles from './styles.module.scss'
+
+class Board extends Component {
+  renderSquare(i) {
+    return (
+      <Square 
+        value={this.props.squares[i]}
+        onClick={() => this.props.onClick(i)}
```

Comment:
Avoid declaring functions within JSX code. This will cause that a new function is created in every render, therefore, if this function is passed down as props, children will re-render unnecessarily. Check [this](https://reactjs.org/docs/faq-functions.html) for reference. You can extract the click handler function to a method of the Board class and then reference it when mapping the component.